sass/components/page.scss

```diff
@@ -0,0 +1,6 @@
+@import "./mixins/page.scss";
+/*start page block  */
+.page{
+    @include create-page($bg-page-color,$content-width:$default-page-width, $footer-width:$default-page-width);
```

Comment:
is there any reason to set with of content and footer but not header. Normally, we do not have this case